language/tools/move-cli/src/package/cli.rs

```diff
@@ -165,6 +165,83 @@ pub enum ProverOptions {
     Options(Vec<String>),
 }
 
+impl CoverageSummaryOptions {
+    pub fn handle_command(&self, config: move_package::BuildConfig, path: &Path) -> Result<()> {
+        let coverage_map = CoverageMap::from_binary_file(path.join(".coverage_map.mvcov"));
+        let package = config.compile_package(path, &mut Vec::new())?;
+        let modules: Vec<_> = package
+            .modules()
+            .filter_map(|unit| match &unit.unit {
+                CompiledUnit::Module(NamedCompiledModule { module, .. }) => Some(module.clone()),
+                _ => None,
+            })
+            .collect();
+        match self {
+            CoverageSummaryOptions::Source { module_name } => {
+                let unit = package
```

Comment:
I found this finding unit pattern to occur quite common. Maybe add that as helper function in the package?